cpp/src/tests/CMakeLists.txt

```diff
@@ -139,6 +139,14 @@ set(PAGERANK_TEST_SRC
 
 ConfigureTest(PAGERANK_TEST "${PAGERANK_TEST_SRC}")
 
+###################################################################################################
+#- native sssp tests -- ---------------------------------------------------------------------------
+set(NATIVE_SSSP_TEST_SRCS
+	"${CMAKE_SOURCE_DIR}/../thirdparty/mmio/mmio.c"
+	"${CMAKE_CURRENT_SOURCE_DIR}/sssp/sssp_test.cu")
+
+ConfigureTest(NATIVE_SSSP_TEST "${NATIVE_SSSP_TEST_SRCS}")
```

Comment:
And once you remove the outdated SSSP_TEST, you can rename this to just SSSP_TEST :-)

Reply:
Done in https://github.com/rapidsai/cugraph/pull/325